storage/mysql/tree_storage.go

```diff
@@ -147,6 +157,9 @@ func (m *mySQLTreeStorage) beginTreeTx(ctx context.Context, tree *trillian.Tree,
 		klog.Warningf("Could not start tree TX: %s", err)
 		return treeTX{}, err
 	}
+	// If the tree description starts with magic constant then we will only write a single
+	// revision for each subtree tile, which simplifies the reading query.
+	subtreeRevs := !strings.HasPrefix(tree.Description, "[NO_SUBTREE_REVISIONS]")
```

Comment:
Should this magic constant be somewhere it can be referenced from here and from the `createTree` cmd? (Top level `/internal/...` pkg?)

Reply:
This constant no longer exists, which is good news all round.